tiledb/sm/fragment/fragment_metadata.cc

```diff
@@ -498,7 +498,8 @@ Status FragmentMetadata::add_max_buffer_sizes_sparse(
   RETURN_NOT_OK(load_rtree(encryption_key));
 
   // Get tile overlap
-  auto tile_overlap = rtree_.get_tile_overlap(subarray);
+  std::vector<bool> is_default(subarray.size(), false);
+  auto tile_overlap = rtree_.get_tile_overlap(subarray, is_default);
```

Comment:
Is there certainty here that the NDRange& subarray is entirely NON-default, and/or does it not matter?

Reply:
There is. This all gets called from Array::get_max_buffer_size, and there is a check in there to not allow var size dimensions. For fixed size dimensions, even if the dimension might be default, there is still values in there representing the non empty domain so we are good.